### ISSUE TITLE
swagger_to_py_client.py: missing operation parameters

### ISSUE THREAD

**Issue by falzm** [Parquery/swagger-to#94]
According to the project [README](https://github.com/Parquery/swagger-to/#missing-features) the *"Python client support query, body, path and header parameters"*, however when generating code using `swagger_to_py_client.py` I end up with methods missing input parameters.

For the following OpenAPI command:

```yaml
components:
  schemas:
    load-balancer:
      type: object
      properties:
        # ...
    operation:
      type: object
      properties:
        # ...

paths:
  /load-balancer:
    post:
      description: ""
      operationId: create-load-balancer
      parameters: []
      requestBody:
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/load-balancer'
        required: true
      responses:
        "200":
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/operation'
          description: "200"
      summary: Create a new Load Balancer
      tags:
      - network-load-balancer
```

The following Python code is generated:

```python
    def create_load_balancer(self) -> bytes:
        """
        Send a post request to /load-balancer.

        :return: 200
        """
        url = self.url_prefix + '/load-balancer'

        resp = self.session.request(method='post', url=url)

        with contextlib.closing(resp):
            resp.raise_for_status()
            return resp.conten
```

Did I miss something?

**Comment by mristin** [comment]
Hi @falzm ,
The parameters in your spec are indeed empty (`parameters: []`)?

**Comment by falzm** [comment]
Hi @mristin, indeed I didn't think of actual parameters but rather of `requestBody` – sorry for the confusion. Do you plan on supporting request body anytime soon?

**Comment by mristin** [comment]
Hi @falzm ,
Are you mixing OpenAPI3 and Swagger spec? I think the body parameters are specified like this:
https://github.com/Parquery/swagger-to/blob/d20be76c62e5c3da718d52fa53c749d1b2926ba0/tests/cases/py_client/general/swagger.yaml#L139

**Comment by mristin** [comment]
@falzm wouldn't that conflict with the parameters? Is the semantic equal as the body parameter?

**Comment by mristin** [comment]
@falzm I had a closer look at the Swagger docs (https://swagger.io/docs/specification/2-0/describing-request-body/). I could not find any mention of `requestBody` -- are you sure that it is a valid Swagger (*i.e.* OpenAPI 2) property?

**Comment by falzm** [comment]
@mristin my bad, I thought this was also available in the Swagger 2.0 spec (aka. OpenAPI 2.0). Well too bad for me, I'll have to find another tool compatible with OAS 3.0 for my use case ¯\_(ツ)_/¯
